runtime/nsd_ptcp.c

```diff
@@ -488,18 +490,18 @@ LstnInit(netstrms_t *pNS, void *pUsr, rsRetVal(*fAddLstn)(void*,netstrm_t*),
 			continue;
 		}
 
-		#ifdef IPV6_V6ONLY
-			if(r->ai_family == AF_INET6) {
-				isIPv6 = 1;
-				int iOn = 1;
-				if(setsockopt(sock, IPPROTO_IPV6, IPV6_V6ONLY,
-				      (char *)&iOn, sizeof (iOn)) < 0) {
-					close(sock);
-					sock = -1;
-					continue;
-				}
+#ifdef IPV6_V6ONLY
```

Comment:
we prefer to have the preprocessor indented with the code - else it sometimes gets very badly readable

Reply:
I don't understand this modification. I am not its author.

Reply:
We need to clarify this. We also need to make sure that all authors explicitely agree to the contubution. If there are multiples, please make them at least post here that they do. Else we run into license issues. If this is a company contribution (where the authors assigned copyright to the company), it would be good to have a company statement here on file.

Reply:
I said that I was not the author of this modification because the view below shows on line 493 the pre-processor directive without indent. But in my PR it is not : code and preprocessor are indent together. It seems to be a github issue affecting the view.

Reply:
No problem, I was the author. I usually not indent preprocessor and I change them by reflex. I rolled back the modification.